CONTRIBUTING.md

```diff
@@ -15,3 +17,29 @@
 * Be descriptive about the problem and reason about the proposed solution
 * Include release notes describing the potential impact of the change in the pull request
 * Make sure there has been at least one approval from Shopify before merging
+
+## Release
```

Comment:
Ah, also the release section should not be in the CONTRIBUTING page. This page is for any people what wants to contribute. People outside Shopify can't release this gem, so it makes no sense for them to read about something they can't do.